### PR TITLE
[Nette 3.0] Fixes #4387 : Add ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRector

### LINKED ISSUE #4387: [Nette 3.0] Add a new rule to convert addUpload() with 3rd argument true to addMultiUpload()
See

```diff
 $form = new Nette\Forms\Form();
-$form->addUpload('...', '...', true);
+$form->addMultiUpload('...', '...');
```

Add to nette30 set: https://github.com/rectorphp/rector/blob/master/config/set/nette-30.php#L26

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/nette/src/Rector/MethodCall/ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRector.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        $args = $node->args;
+        if (! isset($args[2])) {
+            return null;
+        }
+
+        $thirdArgumentValue = $node->args[2]->value;
+        if (
+            $thirdArgumentValue instanceof ConstFetch
+            && count($thirdArgumentValue->name->parts) === 1
+            && $thirdArgumentValue->name->parts[0] === 'true'
```
Comment (TomasVotruba) [inline rules/nette/src/Rector/MethodCall/ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRector.php:69]: Here could be used `$this->isTrue(...)` method

**Review by TomasVotruba — COMMENTED** [review]
I've left a comment

File: rules/nette/src/Rector/MethodCall/ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRector.php (hunk 1 above)
Comment (samsonasik) [reply rules/nette/src/Rector/MethodCall/ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRector.php]: implemented.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thanks!

**Pull request merged**